sources/documents/RTB/Verbali/Interni/VerbaleInterno_2024-04-20/capitolo-riunione.tex

```diff
@@ -0,0 +1,87 @@
+\section{Riunione}
+\subsection{Ordine del giorno}
+\begin{itemize}
+	\item Raccolta dei risultati del questionario e valutazione dello \glossario{sprint} appena concluso;
+  \item Proposte di miglioramento per il prossimo \glossario{sprint} e definizione di un piano d'azione;
+  \item Pianificazione dell'iterazione successiva;
+  \item Resoconto dell'\AdR;
+  \item Discussione sul \Gls;
+  \item Analisi degli \glossario{ITS} individuati come alternative a \glossario{GitHub}.
+\end{itemize}
+
+\subsection{Discussione e decisioni}
+
+\subsubsection{Valutazione del primo sprint}
+\par Di seguito sono riportati i risultati del questionario di valutazione dello \glossario{sprint}, realizzato dal responsabile per ottimizzare la fase di \glossario{retrospettiva}:
+\begin{itemize}
```

Comment:
forse raggrupperei in testa all'eenco tutti i punti con una valutazione e a seguire le note rilevate

Reply:
ok